sources.yml

```diff
@@ -12,9 +12,6 @@
 
 # Generally organized by speed
 
-- name: create
-  script: ome_zarr_create
```

Comment:
Why did this get removed, @will-moore ?

Reply:
It's duplicated on the next 2 lines. I assumed that was not intended?

Reply:
Wow. Ok. Thanks. Maybe I introduced when rebasing? No clue.